src/DataTable/Table.jsx

```diff
@@ -3,24 +3,33 @@ import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import TableHeaderRow from './TableHeaderRow';
 import TableRow from './TableRow';
+import { useRows } from './hooks';
```

Comment:
Abstracting this out might not be needed if a generic TableRows (ItemsList) component could be made

Reply:
Also going to go on my list for later improvements. 😉